admin/sql/updates/2021-04-12-migrate-existing-spotify-users-to-new-schema.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
```

Comment:
what will the process be when we deploy this? I suspect we'll need to re-run this script in order to add all of the new users that have since configured spotify with the old system, right? Perhaps then it might be a good idea to delete from this table and reset the sequence before doing the insert? then we can just reuse this script again when we do the final deploy